resources/views/livestreams/index.blade.php

```diff
@@ -23,7 +23,9 @@
                 <h1 class="osu-page-header__title">{{ trans('livestreams.top-headers.headline') }}</h1>
 
                 <p class="osu-page-header__title osu-page-header__title--smaller">
-                    {{ trans('livestreams.top-headers.description') }}
+                    {!! trans('livestreams.top-headers.description', [
+                        'here' => '<a href="'.e(wiki_url('Guides/Live_Streaming_osu!')).'">'.e(trans('wiki.show.translation.here')).'</a>',
```

Comment:
can use `Html::link()` instead; also the translation key is wrong

Reply:
Is changing the structure to Html::link()  or i dont have to do it?

Reply:
`Html::link` (or `link_to`) to generate the link tag instead of stringing together `<a href...`.
The function also escapes the text automatically